public/js/webcam.js

```diff
@@ -95,17 +101,24 @@ tryAgainButton.addEventListener("click", () => {
 });
 
 function hideAndShowHTMLElementsForTryAgainButton() {
-    video.style.display = "block";
-    takePhotoButton.style.display = "block";
-    tryAgainButton.style.display = "none";
-    getTracksButton.style.display = "none";
-    canvas.style.display = "none";
+    video.className = "video-no-animation";
+    show([video, beforeCaptureGroup]);
+    hide([afterCaptureGroup, canvas]);
 }
 
 function hideAndShowHTMLElementsForCaptureButton() {
-    video.style.display = "none";
-    takePhotoButton.style.display = "none";
-    tryAgainButton.style.display = "block";
-    getTracksButton.style.display = "block";
-    canvas.style.display = "block";
+    hide([video, beforeCaptureGroup]);
+    show([afterCaptureGroup, canvas]);
+}
+
+function hide(elements) {
+    for (let element of elements) {
+        element.style.setProperty("display", "none");
+    }
+}
+
+function show(elements) {
+    for (let element of elements) {
+        element.style.removeProperty("display");
```

Comment:
Does removing it set the display to block?

Reply:
Oh I believe it resets it to whatever default display that bootStrap uses, it could be flex, block, inline etc